src/main/resources/content/k8s/do-not-run-containers-as-root.json

```diff
@@ -1 +1,8 @@
-{"severity":"high","rule":"$.spec.template.spec.securityContext.runAsUser < 1","id":"314eba46-a376-43f6-9a0a-8517818301f1","enabled":true,"resourceType":"k8s","policy":"do not run containers as root"}
\ No newline at end of file
+{
+  "severity": "high",
+  "rule": "$.spec.template.spec.securityContext.runAsUser < 1",
+  "id": "314eba46-a376-43f6-9a0a-8517818301f1",
+  "enabled": true,
+  "resourceType": "k8s",
+  "policy": "do not run containers as root"
```

Comment:
description field missing here as well.
Also, the policy name is not sentence case.